pkg/plugins/models.go

```diff
@@ -18,16 +18,16 @@ type PluginNotFoundError struct {
 }
 
 func (e PluginNotFoundError) Error() string {
-	return fmt.Sprintf("plugin with ID %q not found", e.PluginID)
+	return fmt.Sprintf("plugin with ID %s not found", e.PluginID)
```

Comment:
Why the change? I think quoting here is better, because the plugin ID could be empty (typical in the case of a bug) or have spaces, in which case an error string that doesn't quote the ID makes for more difficult debugging.

Reply:
I changed this as the quotes end up getting escaped in the logs so it makes it harder to read. Maybe single quoting is the better option in this case?